#goal definition
int32 reps
---
#result definition
int32 done
---
#feedback
float32 fb_x
float32 fb_y
float32 fb_z
